manifest.json

```diff
@@ -25,40 +30,21 @@
         "install" : [
             {
                 "name": "domain",
-                "type": "domain",
-                "ask": {
-                    "en": "Choose a domain for pgAdmin",
-                    "fr": "Choisissez un domaine pour pgAdmin"
-                },
-                "example": "domain.org"
+                "type": "domain"
             },
             {
                 "name": "path",
                 "type": "path",
-                "ask": {
-                    "en": "Choose a path for pgAdmin",
-                    "fr": "Choisissez un chemin pour pgAdmin"
-                },
                 "example": "/pgadmin",
                 "default": "/pgadmin"
             },
             {
                 "name": "admin",
-                "type": "user",
-                "ask": {
-                    "en": "Choose the only allowed admin user",
-                    "fr": "Choisissez l'unique utilisateur autorisé"
-                },
-                "example": "johndoe"
+                "type": "user"
             },
             {
-                "name": "admin_password",
-                "type": "password",
-                "ask": {
-                    "en": "Enter a password for the administrator",
-                    "fr": "Entrez un mot de passe pour l'administrateur"
-                },
-                "example": "**Sup3rS3cr3t**"
+                "name": "password",
```

Comment:
Why do you remove the password Here ?

We still need it !!!!

Reply:
Password is not removed, I just change the name of the variable from `admin_password` to `password`

Reply:
Ah yes didn't get it